src/index.html

```diff
@@ -1,12 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" >
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-monday calendar--month-length-31">
```

Comment:
I think instead of `monday` in should be `mon`  according to your variables
![image](https://github.com/mate-academy/layout_calendar/assets/91946036/386a64a3-b723-4ea8-9652-28be495f4442)